src/common/tusb_fifo.c

```diff
@@ -148,27 +148,34 @@ bool tu_fifo_read(tu_fifo_t* f, void * buffer)
 /******************************************************************************/
 uint16_t tu_fifo_read_n (tu_fifo_t* f, void * buffer, uint16_t count)
 {
-  if( tu_fifo_empty(f) ) return 0;
+  if(tu_fifo_empty(f))
```

Comment:
I'm not that familiar with TinyUSB style guidelines, but one-liners are pretty common so I guess that they are preferred, and in this case this line and next change should not be here at all.